enzy_htp/structure/structure_translator/api.py

```diff
@@ -0,0 +1,70 @@
+"""Free function for translating the names of a Structure()'s Residue() and Atom() objects. Should be called
+by users, supports the conversion of names between EnzyHTP standard (AmberMD) and:
+
+    + Rosetta
+
+as well as converting back to EnzyHTP standard. All functionality is handled through `translate_structure()`.
+
+Author: Chris Jurich <chris.jurich@vanderbilt.edu>
+Date: 2024-02-11
+"""
+from typing import Dict, Union
+
+from enzy_htp.core import _LOGGER
+
+from ..atom import Atom
+from ..residue import Residue
+from ..structure import Structure
+
+from .translator_base import TranslatorBase
+from .rosetta_translator import RosettaTranslator
+
+
+def translate_structure(stru:Structure, start_naming:str=None, end_naming:str=None) -> Union[None, Structure]:
+    """Translates names of the supplied Structure()'s Residue() and Atom() objects to EnzyHTP standard or 
+    a supported naming convention. Exactly one of start_naming or end_naming scheme must be supplied. It is assumed that
+    the other, blank parameter is EnzyHTP standard/AmberMD. end_naming should be left blank if a Structure() is to be 
+    converted to EnzyHTP standard and start_naming should be left blank if a Structure() is already in EnzyHTP standard naming.
+    Function performs basic checks on supplied naming parameters and will error if they are invalid. Naming is always done
+    in place.
+
+    Args:
+        stru: The Structure() to rename.
+        start_naming: The starting naming scheme as a str().
+        end_naming:  The ending naming scheme as a str().
+
+    Returns:
+        Nothing or a renamed Structure(), depending on usage of inplace. 
```

Comment:
Seems inplace is missing

Reply:
Currently all changes are made in place

Reply:
Thanks for adding this. I changed it so it is all in place, without an option to do a copy. I have updated the documentation to reflect that.